dist/pom.xml

```diff
@@ -238,10 +264,10 @@
             <phase>package</phase>
             <configuration>
               <target>
-                <copy failonerror="${zbctl.force}" file="${zbctl.rootDir}/clients/go/cmd/zbctl/dist/zbctl" tofile="${project.build.directory}/camunda-cloud-zeebe/bin/zbctl"></copy>
+                <chmod dir="${project.build.directory}/camunda-cloud-zeebe/bin" failonerror="${zbctl.force}" includes="zbctl*" perm="ugo+rx"></chmod>
```

Comment:
:x: I think this change comes from sortpom and, to me at least, looks quite dangerous. The `chmod` needs to run after copying all the binaries are copied.